FirebaseRemoteConfig/Tests/Unit/RCNRemoteConfigTest.m

```diff
@@ -1464,6 +1500,48 @@ - (void)testConfigureConfigWithValidInput {
   XCTAssertEqual([config valueForKey:@"_appName"], kFIRDefaultAppName);
 }
 
+#pragma mark - Realtime tests
+
+- (void)testRealtimeAddConfigUpdateListenerWithValidListener {
+  for (int i = 0; i < RCNTestRCNumTotalInstances; i++) {
+    OCMStub([_configRealtime[i] beginRealtimeStream]).andDo(nil);
+
+    id completion = ^void(NSError *_Nullable error) {
+      if (error != nil) {
```

Comment:
hmm is this block async? If it is you will need to setup XCTestExpectation to ask the test to wait until the async method is finished so that it won't finish too early to cause test to fail:

https://developer.apple.com/documentation/xctest/asynchronous_tests_and_expectations?language=objc

Reply:
Oh that makes sense, I updated the tests to do this.